extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -845,4 +857,57 @@ export class AmpA4A extends AMP.BaseElement {
       });
     });
   }
+
+  /**
+   * @param {!Window} iframeWin
+   */
+  registerExpandUrlParams_(iframeWin) {
+    iframeWin.document.documentElement.addEventListener('click',
```

Comment:
I added some comments to this respect. We do need the 2 handlers for correctness, because one is a capture phase handler and thus reliably runs first.